.github/workflows/integration-test-workflow.yaml

```diff
@@ -0,0 +1,50 @@
+name: Integration Test Workflow
+
+on:
+  workflow_call:
+    secrets:
+      WAPM_DEV_TOKEN:
+        required: true
+
+jobs:
+  run-unspecified-tests:
+    name: Run root tests
+    runs-on: ubuntu-24.04
+
+    steps:
+      - name: Clone repository
+        uses: actions/checkout@v4
+        with:
+          submodules: true
+      - uses: wasmerio/setup-wasmer@v2
+      - name: Install Deno
+        uses: denoland/setup-deno@v2
+        with:
+          deno-version: v2.x
+      - name: Test
+        env:
+          WASMER_REGISTRY: https://registry.wasmer.wtf/graphql
+          WASMER_TOKEN: ${{ secrets.WAPM_DEV_TOKEN }}
+        run: ./run.sh
+
+  run-logvalidation-tests:
+    name: Run log tests
+    runs-on: ubuntu-24.04
+
+    steps:
+      - name: Clone repository
+        uses: actions/checkout@v4
+        with:
+          submodules: true
+      - uses: wasmerio/setup-wasmer@v2
+      - name: Install Deno
+        uses: denoland/setup-deno@v2
+        with:
+          deno-version: v2.x
+      - name: Test
+        env:
+          WASMER_REGISTRY: https://registry.wasmer.wtf/graphql
+          WASMER_TOKEN: ${{ secrets.WAPM_DEV_TOKEN }}
+        run: |
+          deno test --filter "Log test: Check fetch is logged on simple logging app" --allow-all --quiet
```

Comment:
Is this a debugging leftover?

I don't see why we'd want a separate job for one test?